### ISSUE TITLE
508-defect-1 [FOCUS]: Feedback button's focus must be visible

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#26022]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Angela and Josh

## User Story or Problem Statement

As a keyboard user, I expect interactive elements to have the va.gov yellow focus halo so I can easily find and track it as I navigate through the page. 

## Details

The focus halo for the feedback button does not meet contrast requirements when placed above the footer. 
Although it meets minimum requirements on a white background, it is hardly detectable because:
- it has a dashed stroke of 1px, which is 2px below the WCAG recommendation of 3px
- it's not consistent with the va.gov yellow focus halo 
- by the time a keyboard user tabs to the feedback button, they'll likely be at the footer since it's the last tab stop on the page which means it will consistently be above a blue background instead of a white background

<img width="615" alt="Screen Shot 2021-06-14 at 3 09 51 PM" src="https://user-images.githubusercontent.com/14154792/121946713-36b03880-cd23-11eb-855c-d79ed7150ec2.png">

## Acceptance Criteria

- [x] Use the default va.gov yellow focus halo

## WCAG or Vendor Guidance (optional)

[G195: Using an author-supplied, highly visible focus indicator](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G195)

**Comment by pjhill** [comment]
I'm having trouble reproducing this issue on [https://www.va.gov/find-locations](https://www.va.gov/find-locations) for example. Has this been addressed already?
![image.png](https://images.zenhubusercontent.com/5d2633e0a6db635dbcbaf8d5/703caac8-897a-4ffb-a92a-002a21e251f2)

**Comment by 1Copenut** [comment]
Thanks @pjhill. I cross-checked your screenshot on VA.gov and am seeing the yellow focus halo. AC is met, going to move this one to closed.
